app/models/form526_submission.rb

```diff
@@ -77,6 +77,7 @@ class Form526Submission < ApplicationRecord
   # MAX_PENDING_TIME aligns with the farthest out expectation given in the LH BI docs,
   # plus 1 week to accomodate for edge cases and our sidekiq jobs
   MAX_PENDING_TIME = 3.weeks
+  ZSF_DD_TAG_SERVICE = 'disability-application'
```

Comment:
Just a heads up it looks like @kylesoskin made this [same change in another open PR](https://github.com/department-of-veterans-affairs/vets-api/pull/18801/files#diff-ada3116cb66cda03a293e8c804592ccb3bc181f3e64689c66fe27d19b0d3cad1R80); not sure if it matters whoever gets it in first wins I guess? Haha